performance/collector/search_examples/delete_index.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
If you would like to document API use RAML file, for now I would suggest just to remove these samples